db/migrate/20190226000001_setting_add_sender_format_agent_name.rb

```diff
@@ -0,0 +1,12 @@
+class SettingAddSenderFormatAgentName < ActiveRecord::Migration[5.1]
+  def up
+    # return if it's a new setup
+    return if !Setting.find_by(name: 'system_init_done')
+
+    setting = Setting.find_by(name: 'ticket_define_email_from')
+    return if !setting
+
+    setting.options[:form][0][:options][:AgentName] = 'Agent Name Name'
```

Comment:
There is one Name too much. Should be:
```suggestion
    setting.options[:form][0][:options][:AgentName] = 'Agent Name'
```